resource/ui/options_dialog.ui

```diff
@@ -23,7 +23,7 @@
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
```

Comment:
Please don't change the default page, QDesigner saves the last page you viewed as the default page.

Reply:
i always forget :pensive: